src/App.tsx

```diff
@@ -17,18 +46,30 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onSelect={setSelectedStatus}
+                onChange={setSelectedQuery}
```

Comment:
The naming convention for event handler props suggests using `onQueryChange` instead of `onChange` to make it more descriptive and specific to the action.